docs/troubleshooting.rst

```diff
@@ -2,6 +2,14 @@
 Troubleshooting
 ===============
 
+Installation failure
+====================
+
+``ModuleNotFoundError: No module named 'Cython'``
+
+pip is failing to install `dd-trace` and complaining about a missing module (Cython).
```

Comment:
```suggestion
pip is failing to install ``ddtrace`` and complaining about a missing module (Cython).
```

Reply:
My bad, thanks for reviewing this and submitting improvements.